src/main/java/fly/wild/wizards/serverside/ServerConnector.java

```diff
@@ -0,0 +1,328 @@
+	private final int INT_KEY_SIZE_VALUE = 2048;
+	private final int INT_VALIDITY_VALUE = 365;
+	private final String STRING_SERVER_VALUE = "default-server";
+	private final String STRING_HTTPS_LISTENER_VALUE = "https";
+	private String genKeyStoreName;
+	private String genKeyManagerName;
+	private String genServerSSLContext;
+	private String padding;
+	private String keyStoreFileName;	
+	private ControllerAddress connectionAddress;
+	private OneWayTLSConfigurationConfiguration oneWayTLSConfiguration;
+	private TLSConfiguration tlsConfiguration;
+	private final int connectionTimeOut = 5000;
+	
+	public ServerConnector(TLSConfiguration tlsConfiguration, OneWayTLSConfigurationConfiguration oneWayTLSConfiguration) {
+		LocalTime localTime = LocalTime.now();
+		UUID uuid=UUID.randomUUID();
+		
+		this.oneWayTLSConfiguration = oneWayTLSConfiguration;
+		this.tlsConfiguration = tlsConfiguration;
+		this.ipAddress = this.tlsConfiguration.getServerIP();
+		
+		this.padding = uuid.toString();
+		
+		this.genKeyStoreName = "wizgenks" + this.padding;
+		this.genKeyManagerName = "wizgenkm" + this.padding;
+		this.genServerSSLContext = "wizgenssl" + this.padding;
+		this.keyStoreFileName = this.oneWayTLSConfiguration.getKeyStoreFileNameValue();
+		this.alias = this.oneWayTLSConfiguration.getFirstAndLastNameValue();		
+		this.distinguishedName = "CN=" + this.oneWayTLSConfiguration.getFirstAndLastNameValue() +
+				";" + "OU=" + this.oneWayTLSConfiguration.getOrganizationalUnitValue() +
+				";" + "O=" + this.oneWayTLSConfiguration.getOrganizationValue() +
+				";" + "L=" + this.oneWayTLSConfiguration.getCityOrLocalityValue() +
+				";" + "ST=" + this.oneWayTLSConfiguration.getStateOrProvinceValue() +
+				";" + "C=" + this.oneWayTLSConfiguration.getCountryCodeValue();
+
+		this.logMessage = new StringBuilder("This is the TLSWizard run of "+localTime.toString()+"\n");
+		connectionAddress = new ControllerAddress("remote+http",ipAddress,9990);	
+		
+	}
+	
+	
+	public boolean configureOneWayTLS() {
+		
+		boolean success = true;
+		
+		// Create a key store		
+		DefaultOperationRequestBuilder keyStoreBuilder = this.buildKeyStoreNode();
+		// Generate a key pair
+		DefaultOperationRequestBuilder generateCertificateBuilder = buildGenerateCertificateBuilder();
+		// Persist the key pair		
+		DefaultOperationRequestBuilder storeCertificateBuilder = buildStoreCertificateBuilder();
+		//Create a key manager
+		DefaultOperationRequestBuilder keyManagerBuilder = buikdKeyManagerBuilder();
+		//Configure SSL Context
+		DefaultOperationRequestBuilder serverSSLContextBuilder = buildServerSSLContextBuilder();
+		
+		try {
+			
+			ModelControllerClient client = getClientWithNoAuth();		
+			final ModelNode keytore = client.execute(keyStoreBuilder.buildRequest());
+			this.logMessage.append("Creating a key store\n");
+			this.logMessage.append(keytore.toString()).append("\n");
+			
+			final ModelNode cert = client.execute(generateCertificateBuilder.buildRequest());
```

Comment:
I think you can put all the buildRequest() calls inside of the methods and return the  ModelNode from all of them